mdtraj/formats/gro.py

```diff
@@ -85,10 +85,13 @@ def load_gro(filename, stride=None, atom_indices=None, frame=None):
         Use this option to load only a single frame from a trajectory on disk.
         If frame is None, the default, the entire trajectory will be loaded.
         If supplied, ``stride`` will be ignored.
+    top : mdtraj.core.Topology, default=None
+        if you give a topology as input the topology won't be parsed from the pdb file
```

Comment:
this should be `gro`
```suggestion
        if you give a topology as input the topology won't be parsed from the gro file
```